src/java/edu/slu/tpen/servlet/CanvasServlet.java

```diff
@@ -100,76 +103,88 @@ public String getServletInfo() {
-                canvasHeight = 0;
-                canvasWidth = 0;
+            else{ //define a 0, 0 storedDims
+                storedDims = new Dimension(0,0);
+            }
+            result.put("width", canvasWidth);
+            result.put("height", canvasHeight);
+            List<Object> images = new ArrayList<>();
+            Map<String, Object> imageAnnot = new LinkedHashMap<>();
+            imageAnnot.put("@type", "oa:Annotation");
+            imageAnnot.put("motivation", "sc:painting");
+            String imageURL = f.getImageURL();
+            if (imageURL.startsWith("/")) {
+                imageURL = String.format("%spageImage?folio=%s",getRbTok("SERVERURL"), f.getFolioNumber());
+            }
+            Map<String, Object> imageResource = buildQuickMap("@id", imageURL, "@type", "dctypes:Image", "format", "image/jpeg");
+
+            if (storedDims.height > 0) { //We could ignore this and put the 0's into the image annotation
+                //doing this check will return invalid images because we will not include height and width of 0.
+               imageResource.put("height", storedDims.height ); 
+               imageResource.put("width", storedDims.width ); 
             }
-      }
-      //We will return 0 for the values here no matter what so that the object returned is valid IIIF from this servlet.
-      //We could do the same check that JsonLDExporter does and return the object without the height and width so it is invalid (on purpose).
-      result.element("width", canvasWidth);
-      result.element("height", canvasHeight);
-      imageResource.element("height",0 ); 
-      imageResource.element("width",0 ); 
-      
-      imageAnnot.element("resource", imageResource);
-      imageAnnot.element("on", canvasID);
-      images.add(imageAnnot);
-      //FIXME this looks for data on anno store.  Cannot build list this way for canvas servlet while not looking to anno store.  
-      otherContent = getAnnotationListsForProject(-1, canvasID, 0);
-      //otherContent = Canvas.getLinesForProject(projID, canvasID, f.getFolioNumber(), u.getUID());
-      //it seems like it wants me to do Arrays.toString(otherContent), but then it is not formatted correctly.  
-      result.element("otherContent", fromObject(otherContent));
-      result.element("images", images);
-      return result;
+            imageAnnot.put("resource", imageResource);
+            imageAnnot.put("on", canvasID);
+            images.add(imageAnnot);
+            //If this list was somehow stored in the SQL DB, we could skip calling to the store every time.
+            //System.out.println("Get otherContent");
+            //System.out.println(projID + "  " + canvasID + "  " + f.getFolioNumber() + "  " + u.getUID());;
+            //otherContent = getAnnotationListsForProject(-, canvasID, 0);
+            otherContent = getLinesForProject(projID, canvasID, f.getFolioNumber(), 0); //Can be an empty array now.
```

Comment:
This does not actually contain the lines.  See the example below.  `resources` is an empty array.

```JSON
{
   "@wid":"http://localhost:8080/TPEN/canvas/13251387",
   "@type":"sc:Canvas",
   "label":"IP_001_001",
   "width":749,
   "height":1000,
   "otherContent":[
      {
         "@id":"http://localhost:8080/TPEN/project/13251387/annotations/13251387",
         "@type":"sc:AnnotationList",
         "label":"http://localhost:8080/TPEN/canvas/13251387 List",
         "proj":13251387,
         "on":"http://localhost:8080/TPEN/canvas/13251387",
         "@context":"http://iiif.io/api/presentation/2/context.json",
         "resources":[
            
         ]
      }
   ],
   "images":[
      {
         "@type":"oa:Annotation",
         "motivation":"sc:painting",
         "resource":{
            "@id":"https://iiif.library.utoronto.ca/image/v2/italianpaleography:IP_001_001/full/full/0/default.jpg",
            "@type":"dctypes:Image",
            "format":"image/jpeg"
         },
         "on":"http://localhost:8080/TPEN/canvas/13251387"
      }
   ]
}
```

Here is that same Canvas returned by the /manifest/ servlet.  Note this is the manifest your servlet should be returning.  Test your code by comparing to the canvases in /TPEN/manifest/7211
```JSON
{
   "@id":"http://localhost:8080/TPEN/canvas/13251387",
   "@type":"sc:Canvas",
   "label":"IP_001_001",
   "width":749,
   "height":1000,
   "otherContent":[
      {
         "@id":"http://localhost:8080/TPEN/project/7211/annotations/13251387",
         "@type":"sc:AnnotationList",
         "label":"http://localhost:8080/TPEN/canvas/13251387 List",
         "proj":7211,
         "on":"http://localhost:8080/TPEN/canvas/13251387",
         "@context":"http://iiif.io/api/presentation/2/context.json",
         "resources":[
            {
               "@id":"http://localhost:8080/TPEN/line/104720616",
               "_tpen_line_id":"line/104720616",
               "@type":"oa:Annotation",
               "motivation":"oad:transcribing",
               "resource":{
                  "@type":"cnt:ContentAsText",
                  "cnt:chars":"This is the first line"
               },
               "on":"http://localhost:8080/TPEN/canvas/13251387#xywh=41,64,629,66",
               "_tpen_note":"",
               "_tpen_creator":11,
               "modified":"2022-09-12 13:21:09.0"
            },
            {
               "@id":"http://localhost:8080/TPEN/line/104720617",
               "_tpen_line_id":"line/104720617",
               "@type":"oa:Annotation",
               "motivation":"oad:transcribing",
               "resource":{
                  "@type":"cnt:ContentAsText",
                  "cnt:chars":"This is the second line"
               },
               "on":"http://localhost:8080/TPEN/canvas/13251387#xywh=41,130,629,27",
               "_tpen_note":"",
               "_tpen_creator":11,
               "modified":"2022-09-12 13:21:12.0"
            },
            {
               "@id":"http://localhost:8080/TPEN/line/104720618",
               "_tpen_line_id":"line/104720618",
               "@type":"oa:Annotation",
               "motivation":"oad:transcribing",
               "resource":{
                  "@type":"cnt:ContentAsText",
                  "cnt:chars":"This is the third line"
               },
               "on":"http://localhost:8080/TPEN/canvas/13251387#xywh=41,157,629,19",
               "_tpen_note":"",
               "_tpen_creator":11,
               "modified":"2022-09-12 13:21:15.0"
            },
            {
               "@id":"http://localhost:8080/TPEN/line/104720619",
               "_tpen_line_id":"line/104720619",
               "@type":"oa:Annotation",
               "motivation":"oad:transcribing",
               "resource":{
                  "@type":"cnt:ContentAsText",
                  "cnt:chars":"This is the fourth line"
               },
               "on":"http://localhost:8080/TPEN/canvas/13251387#xywh=41,176,629,16",
               "_tpen_note":"",
               "_tpen_creator":11,
               "modified":"2022-09-12 13:21:18.0"
            },
            {
               "@id":"http://localhost:8080/TPEN/line/104720620",
               "_tpen_line_id":"line/104720620",
               "@type":"oa:Annotation",
               "motivation":"oad:transcribing",
               "resource":{
                  "@type":"cnt:ContentAsText",
                  "cnt:chars":"This is the fifth line"
               },
               "on":"http://localhost:8080/TPEN/canvas/13251387#xywh=41,193,629,31",
               "_tpen_note":"",
               "_tpen_creator":11,
               "modified":"2022-09-12 13:21:21.0"
            },
            {
               "@id":"http://localhost:8080/TPEN/line/104720621",
               "_tpen_line_id":"line/104720621",
               "@type":"oa:Annotation",
               "motivation":"oad:transcribing",
               "resource":{
                  "@type":"cnt:ContentAsText",
                  "cnt:chars":""
               },
               "on":"http://localhost:8080/TPEN/canvas/13251387#xywh=41,224,629,232",
               "_tpen_note":"",
               "_tpen_creator":11,
               "modified":"2022-09-12 11:35:20.0"
            }
         ]
      }
   ],
   "images":[
      {
         "@type":"oa:Annotation",
         "motivation":"sc:painting",
         "resource":{
            "@id":"https://iiif.library.utoronto.ca/image/v2/italianpaleography:IP_001_001/full/full/0/default.jpg",
            "@type":"dctypes:Image",
            "format":"image/jpeg"
         },
         "on":"http://localhost:8080/TPEN/canvas/13251387"
      }
   ]
}
```